live-site/components/judging-section/judging-schedule-hackers/JudgingScheduleHackers.tsx

```diff
@@ -1,8 +1,74 @@
-import React from 'react';
-import JudgingScheduleWelcome from '../judging-schedule-welcome/JudgingScheduleWelcome';
+import React, { useState } from 'react';
+import hackerList from '../../../../judging-algorithm/hackers.json';
+import { HackerEntryType } from '../../../lib/types';
+import {
+  StyledBackButton,
+  StyledBackButtonContainer,
+  StyledJudgesDropdownContainer,
+  StyledJudgesDropdownWrapper,
+  StyledJudgingScheduleSection,
+  StyledP,
+  StyledWelcomePerson
+} from '../judging-schedule-welcome/JudgingScheduleWelcome.styles';
 
 const JudgingScheduleHackers: React.FC = () => {
-  return <JudgingScheduleWelcome schedulePersonType={'Hacker'} />;
+  const projectOptions = [
+    'Select your project',
+    ...hackerList.map((obj: HackerEntryType) => obj['project'])
+  ];
+  const [projectSelected, setProjectSelected] = useState<string>(
+    'Select your project'
+  );
+
+  const [hackerSelected, setHackerSelected] = useState<HackEntryType>(null);
+
+  function getProjectData(projectName: string): HackerEntryType {
+    const hackerObj = hackerList.filter(
+      (hackerObj: HackerEntryType) => hackerObj.project === projectName
+    );
+    return hackerObj[0];
+  }
+
+  function commaSeparatedJudges(judgeList: string[]): string[] {
+    for (let i = 0; i < judgeList.length; i++) {
+      if (i != judgeList.length-1 && !judgeList[i].includes(',')) {
+        judgeList[i] = judgeList[i].concat(', ');
+      }
+    }
+    return judgeList;
+  }
+
+  return (
+    <div>
+      <StyledBackButtonContainer>
+        <StyledBackButton isSmallPrimary={true} btnText="back" btnLink="../" />
+      </StyledBackButtonContainer>
+      <StyledJudgingScheduleSection>
+        <StyledWelcomePerson>Welcome, Hacker!</StyledWelcomePerson>
+        <StyledJudgesDropdownContainer>
+          <StyledJudgesDropdownWrapper
+            id="position-filter"
+            onChange={(e): void => {
+              setProjectSelected(e.target.value);
+              setHackerSelected(getProjectData(e.target.value));
+            }}
+          >
+            {projectOptions.map((curProject: string) => (
+              <option value={curProject} key={curProject}>
+                {curProject}
+              </option>
+            ))}
+          </StyledJudgesDropdownWrapper>
+        </StyledJudgesDropdownContainer>
+        {projectSelected != 'Select your project' && 
+        <>
+        <StyledP>{hackerSelected.time}</StyledP>
+        <StyledP>{commaSeparatedJudges(hackerSelected.judges)}</StyledP>
```

Comment:
Also, can we make the texts left aligned, bigger, the brown color, and add more padding to the top so its not so close to the dropdown?

Reply:
Made the change!
Text center-aligned seemed to be the only way to make it be in the center of the page tho, is that fine?

Reply:
okay!!! this looks fine1